techmatters/flex-plugins
# CHI-2361: Frontload assigning chat channel & service to contacts on creation

## Description

So we can't try to retrieve transcripts for contacts before they have been assigned

### Checklist

### Related Issues
Fixes CHI-2361

### Verification steps

Accept a chat contact
DO NOT complete it yet, or in fact input any data for the contact
Verify the service sid and channel are set for the new contract in the DB
